### ISSUE TITLE
test_blocking_pool_creation_rules cant find Actions three dots btn

### ISSUE THREAD

**Issue by DanielOsypenko** [red-hat-storage/ocs-ci#10115]
Failed to find the element (css selector,button[aria-label="Actions"])
https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#ocs/launches/678/23111

![image](https://github.com/user-attachments/assets/4537d4ca-d74f-45a0-8d91-31feef61e58e)

**Comment by DanielOsypenko** [comment]
issue repeats - https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#ocs/launches/678/23234/1117666/1117747/log?logParams=history%3D1117747%26page.page%3D1

**Comment by DanielOsypenko** [comment]
The rule has changed and now consecutive dots are allowed.
Redo this method `check_edit_labels`:
         
      Labels must start and end with an alphanumeric character,
               can consist of lower-case letters, numbers and non-consecutive dots (.),
               and hyphens (-), forward slash (/), underscore(_) and equal to (=)
